internal/driver/mobile/app/GoNativeActivity.java

```diff
@@ -27,6 +27,11 @@
 import android.widget.TextView;
 import android.widget.TextView.OnEditorActionListener;
 
+// GoNativeActivity is the java implementation that helps Go map to android via the NDK.
```

Comment:
I'm not sure this first sentence is right - the activity is the entry point for our Android boot loader.
The NDK connects to this via C - so it's not really providing any connections itself I don't think?...

Reply:
What I meant (from my VERY limited understanding) is that it was the java SDK object that was a utility for Go to access via the C and the NDK.
I guess I don't know how to express it correctly :)

Reply:
Go accesses the Java object via NDK. The Java provides us certain functionality which is accessed through JNI and C functionality.
Is that clearer?